ddtrace/appsec/_remoteconfiguration.py

```diff
@@ -111,6 +113,7 @@ def _appsec_rules_data(features: Mapping[str, Any], test_tracer: Optional[Tracer
         _add_rules_to_list(features, "rules", "Datadog rules", ruleset)
         _add_rules_to_list(features, "exclusions", "exclusion filters", ruleset)
         _add_rules_to_list(features, "rules_override", "rules override", ruleset)
+        _add_rules_to_list(features, "scanners", "scanners", ruleset)
```

Comment:
Could we add more tests and validate Remote Config payloads with "scanners"?

Reply:
There is no support on the backend for this feature now. This is to make the tracer ready for tests when it will be, but it does nothing right now.

Reply:
ok! :+1: